modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss

```
@@ -74,6 +74,9 @@
 		background-color: $success-l2;
 		border: 1px solid $success-l1;
 		border-radius: 0.25rem;
+		& .btn-secondary {
+			background-color: unset;
+		}
 	}
```

Just need to add spaces

```suggestion
		border-radius: 0.25rem;
		
		&.btn-secondary {
			background-color: unset;
		}
	}
```

---

Thank @kevenleone, I updated the code.